create/mysql.go

```diff
@@ -0,0 +1,98 @@
+package create
+
+import (
+	"context"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+
+	runtimev1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	infra "github.com/ninech/apis/infrastructure/v1alpha1"
+	meta "github.com/ninech/apis/meta/v1alpha1"
+	storage "github.com/ninech/apis/storage/v1alpha1"
+	"github.com/ninech/nctl/api"
+)
+
+type mySQLCmd struct {
+	Name                  string                                 `arg:"" default:"" help:"Name of the MySQL instance. A random name is generated if omitted."`
+	Location              string                                 `default:"nine-cz41" help:"Location where the MySQL instance is created."`
+	MachineType           infra.MachineType                      `help:"Defines the sizing for a particular MySQL instance." placeholder:"nine-standard-1" default:"nine-standard-1"`
+	AllowedCidrs          []storage.IPv4CIDR                     `help:"Specify the allowed IP addresses, connecting to the instance." placeholder:"0.0.0.0/0"`
+	SSHKeys               []storage.SSHKey                       `help:"Contains a list of SSH public keys, allowed to connect to the db server, in order to up-/download and directly restore database backups."`
```

Comment:
Weird, it's working on production...
```
dirk@etkbox ~/nctl $ ./nctl create mysql
 ✓ created mysql "hip-beef" 🏗
```

Reply:
So it seems there is some validation difference between k3s (which we use on staging) and upstream k8s. On the wire, `nctl` sends something like this when executing the above commands:

```json
{"metadata":{"name":"emerging-reaper","namespace":"test","creationTimestamp":null},"spec":{"writeConnectionSecretToRef":{"name":"mysql-emerging-reaper","namespace":"test"},"forProvider":{"machineType":"nine-standard-1","location":"nine-cz41","allowedCIDRs":null,"sshKeys":null,"characterSet":{}}},"status":{"atProvider":{}}}
```

It seems the `null` values for `allowedCIDRs` and `sshKeys` get handled differently by the different environments. 

Sending `null` happens because kong leaves the variables "uninitialized" (so they are `nil`) if you don't set the corresponding arguments. As a workaround, could we have something like:

```go
allowedCIDRs := []storage.IPv4CIDR{}
if cmd.AllowedCidrs != nil {
    allowedCIDRs = cmd.AllowedCidrs
}
// here the same for cmd.SSHKeys
...
```

?

We can then use these variables in `storage.MysqlParameters`.